spec/support/aio.rb

```diff
@@ -0,0 +1,9 @@
+aio = on_supported_os.reject do |os, facts|
+  ['FreeBSD', 'DragonFly', 'Windows'].include?(facts[:operatingsystem])
```

Comment:
Archlinux should get added here, see 75d0369

Reply:
Debatable. It's not really AIO because it uses the system ruby and OS packages, but follows some paths from the AIO packaging. cc @bastelfreak

Reply:
Our params.pp says `$aio_package = ($::osfamily == 'Windows' or $::rubysitedir =~ /\/opt\/puppetlabs\/puppet/)`, following that logic it's not. ;)
`

Reply:
Right, this was reject, not filter :)

Reply:
hey,
we build Puppet against system ruby 2.5.1 at the moment, so `$fact['rubysitedir']` resolves to `/usr/lib/ruby/site_ruby/2.5.0`. Besides that we try to follow the paths in the AIO packages as good as possible.

Reply:
Updated. I think it should now cover Arch. I do wonder: who owns `/etc/puppetlabs/puppet` in packaging? In AIO it's root (which I agree with) but our current module sets it to puppet.